src/content/snapshotOptions/disable-snapshots.mdx

```diff
@@ -41,13 +33,6 @@ const meta: Meta<typeof NotFound> = {
 
 export default meta;
 type Story = StoryObj<typeof NotFound>;
```

Comment:
@domyen can you introduce the story back without the play function so that we have a minimal viable test documented? Effectively turning this into the following:
```md
// Adjust this import to match your framework (e.g., nextjs, vue3-vite)
import type { Meta, StoryObj } from "@storybook/your-framework";
import { NotFound } from "./NotFound";

const meta: Meta<typeof NotFound> = {
  component: NotFound,
  title: "NotFound",
  parameters: {
    // Disables Chromatic's snapshotting on a component level
    chromatic: { disableSnapshot: true },
  },
};

export default meta;
type Story = StoryObj<typeof NotFound>;

export const Default: Story = {};
```